core/external_search.py

```diff
@@ -1546,17 +1546,17 @@ def __init__(self, query_string, filter=None, use_query_parser=True):
             # run the 'fuzzy' hypotheses at all.
             self.fuzzy_coefficient = 0
 
-    def build(self, elasticsearch, pagination=None):
-        """Make an Elasticsearch-DSL Search object out of this query.
+    def build(self, opensearch, pagination=None):
```

Comment:
Looks like the param is changing here. Happy to see it change, but just wanted to make sure we searched for usages of this function that might be using the old param name.

Reply:
Within the function the variable has been refactored and the calling methods don't use an elasticsearch specific name